example/sample_data/sample_column_usage.csv

```diff
@@ -1,13 +1,16 @@
 database,cluster,schema,table_name,column_name,user_email,read_count
-hive,gold,test_schema,test_table1,col1,roald.amundsen@example.org,100
-hive,gold,test_schema,test_table3,col1,aoald0@example.org,10
-hive,gold,test_schema,test_table3,col1,boald1@example.org,10
-hive,gold,test_schema,test_table3,col1,coald2@example.org,10
-hive,gold,test_schema,test_table3,col1,doald3@example.org,10
-hive,gold,test_schema,test_table3,col1,eoald4@example.org,10
-hive,gold,test_schema,test_view1,col1,foald5@example.org,10
-hive,gold,test_schema,test_view1,col1,goald6@example.org,10
-hive,gold,test_schema,test_table2,col1,hoald7@example.org,10
-hive,gold,test_schema,test_table2,col1,ioald8@example.org,10
-hive,gold,test_schema,test_table2,col1,joald9@example.org,10
+hive,gold,test_schema,test_table1,col1,roald.amundsen@example.org,500
+hive,gold,test_schema,test_table1,col1,aoald0@example.org,100
```

Comment:
is it mainly usage is the change?

Reply:
Yeah sorry about this. This should be in its own MR probably. I noticed when testing that popular tables no longer show up, I believe this is due to requirement that there are minimum 10 unique users. I was lazy and thought I would add it to this MR as well.